### PR TITLE
Eagle Optimized TensorFlow

### PR DESCRIPTION
**Pull request by PaulMDiaz**
Hello,

I am submitted this pull request for my upcoming tutorial on optimized TensorFlow builds for eagle.

Thank you!
-Paul Diaz

### CODE REVIEW

File: workshops/Optimized_TF/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Install instructions for GPU use
+
+1. Navigate to your eagle home directory or scratch directory using
```
Comment (KevinSayers) [inline workshops/Optimized_TF/README.md:3]: eagle -> Eagle

File: workshops/Optimized_TF/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      module load gcc/7.4.0
+      module load cuda/10.0.130
+      module load cudnn/7.4.2/cuda-10.0
+      module load conda
+      ```
+  * b) build a predefined conda environment
+      ```
+      conda env create -f py38tf23.yml
+      ```
+  * c) Active the conda environment
```
Comment (KevinSayers) [inline workshops/Optimized_TF/README.md:62]: Active -> Activate

File: workshops/Optimized_TF/README.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      ```
+  * d) Install the precompiled TensorFlow installation from a wheel
+      ```
+      pip install --upgrade --no-deps --force-reinstall /nopt/nrel/apps/wheels/tensorflow-2.2.1-cp37-cp37m-linux_x86_64.whl
+      ```
+      or
+      ```
+      pip install --upgrade --no-deps --force-reinstall /nopt/nrel/apps/wheels/tensorflow-2.2.1-cp37-cp37m-linux_x86_64.whl
+      ```
+  * e) If you are an allocated or interactive node you can test the install by running
```
Comment (KevinSayers) [inline workshops/Optimized_TF/README.md:102]: If you are an -> If you are on an

File: workshops/Optimized_TF/README.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+  * c)  Active the conda environment
+      ```
+      source activate py37tf20
+      ```
+  * d) Install the precompiled TensorFlow installation from a wheel
+      ```
+      pip install --upgrade --no-deps --force-reinstall /nopt/nrel/apps/wheels/tensorflow-2.0.0-cp37-cp37m-linux_x86_64.whl
+      ```
+
+  * e) If you are an allocated or interactive node you can test the install by running
```
Comment (KevinSayers) [inline workshops/Optimized_TF/README.md:131]: If you are an -> If you are on an

**Review by KevinSayers — COMMENTED** [review]
A couple minor grammatical suggestions

File: workshops/Optimized_TF/README.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+```
+cd ~/
+```
+or
+```
+cd /scratch/$USER/
+```
+2. Clone the github repo into the directory you chose
+```
+git clone https://github.nrel.gov/hpc-apps/Optimized_TF
```
Comment (KevinSayers) [inline workshops/Optimized_TF/README.md:13]: Update for nrel/hpc repo instead

File: workshops/Optimized_TF/README.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+```
+3. Navigate to the repo
+```
+cd ./Optimized_TF/
```
Comment (KevinSayers) [inline workshops/Optimized_TF/README.md:17]: update based on the different repo being cloned

**Review by KevinSayers — COMMENTED** [review]
repo changes

File: workshops/Optimized_TF/README.md (hunk 5 above)
Comment (diskwarrior) [reply workshops/Optimized_TF/README.md:13]:
```suggestion
https://github.com/NREL/HPC.git
```

File: workshops/Optimized_TF/README.md (hunk 6 above)
Comment (diskwarrior) [reply workshops/Optimized_TF/README.md:17]:
```suggestion
cd ./HPC/workshops/Optimized_TF/
```

**Review by diskwarrior — APPROVED** [review]
This content looks great. Thanks Paul!

**Pull request merged**